archunit/src/main/java/com/tngtech/archunit/core/domain/JavaCodeUnit.java

```diff
@@ -359,6 +370,30 @@ private Get() {
                     }
                 };
             }
+
+            /**
+             * @see #getCallsOfSelf()
+             */
+            @PublicAPI(usage = ACCESS)
+            public static final ChainableFunction<JavaCodeUnit, Set<? extends JavaAccess<?>>> GET_CALLS_TO_SELF =
```

Comment:
By convention the constant name should be the exact method name in upper case. I.e. if the getter is `getCallsOfSelf` then the function should be `GET_CALLS_OF_SELF`

Reply:
Corrected.